dishonored2.asl

```diff
@@ -117,7 +138,7 @@ update
         // Check if inside starting area.
             posX-delta < current.x && current.x < posX+delta &&
             posY-delta < current.y && current.y < posY+delta &&
-            posZ-delta < current.z && current.z < posZ+delta;
+            posZ-delta < current.z && current.z < posZ+2;
```

Comment:
I'd originally found the cam offsets at a static memory location, but obviously the cam reports its position as a bit higher than the character itself.  The expanded range allowed the check to work with either player or cam co-ords with no obvious drawbacks to either.

After getting it to work with campos, I found the playerpos offsets, so this can be tightened back up if desired.  Though campos might be simpler to find if the exe ever gets another update.

Thoughts?    Tighten back up, or leave functional with either?